### PR TITLE
3914 Validate Docket Record Sequence

### PR DESCRIPTION
**Pull request by wbyoko**

### CODE REVIEW

File: web-client/src/presenter/actions/EditDocketRecordEntry/primePropsFromEditDocketRecordEntryModalAction.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { state } from 'cerebral';
+
+/**
+ * update props from modal state to pass to through sequence
```
Comment (matthopson) [inline web-client/src/presenter/actions/EditDocketRecordEntry/primePropsFromEditDocketRecordEntryModalAction.js:4]: The wording on this is a little funny. Probably easier to address on the next branch.

File: web-client/src/presenter/actions/EditDocketRecordEntry/validateDocketRecordAction.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/**
+ * validates the petition.
```
Comment (matthopson) [inline web-client/src/presenter/actions/EditDocketRecordEntry/validateDocketRecordAction.js]: Validates the Docket Record

File: web-client/src/presenter/actions/EditDocketRecordEntry/validateDocketRecordAction.test.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      }),
+    };
+
+    presenter.providers.path = {
+      error: errorStub,
+      success: successStub,
+    };
+  });
+
+  it('should call the path success when no errors are found', async () => {
```
Comment (matthopson) [inline web-client/src/presenter/actions/EditDocketRecordEntry/validateDocketRecordAction.test.js]: Maybe "should call the success path" ?

File: web-client/src/presenter/actions/EditDocketRecordEntry/validateDocketRecordAction.test.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      },
+      state: {
+        form: mockDocketRecord,
+      },
+    });
+
+    expect(successStub.calledOnce).toEqual(true);
+  });
+
+  it('should call the path error when any errors are found', async () => {
```
Comment (matthopson) [inline web-client/src/presenter/actions/EditDocketRecordEntry/validateDocketRecordAction.test.js]: Maybe "should call the error path" ?

File: web-client/src/presenter/actions/EditDocketRecordEntry/validateDocketRecordAction.js (hunk 2 above)
Comment (kkoskelin) [inline web-client/src/presenter/actions/EditDocketRecordEntry/validateDocketRecordAction.js]: jsdoc

**Review by matthopson — APPROVED** [review]
Couple of small nits that are easily addressed on the next PR if you want.

File: web-client/src/presenter/actions/EditDocketRecordEntry/validateDocketRecordAction.js (hunk 2 above)
Comment (wbyoko) [reply web-client/src/presenter/actions/EditDocketRecordEntry/validateDocketRecordAction.js]: ?

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**